app/models/plan.rb

```diff
@@ -420,7 +427,7 @@ def owner
                   .administrator
                   .order(:created_at)
                   .pluck(:user_id).first
-    User.find(usr_id)
+    usr_id.present? ? User.find(usr_id) : nil
```

Comment:
this was an edge case error that came up with some test records (a plan with no owner)